config-model/src/main/java/com/yahoo/searchdefinition/RankProfile.java

```diff
@@ -54,36 +54,54 @@
 
     public final static String FIRST_PHASE = "firstphase";
     public final static String SECOND_PHASE = "secondphase";
-    /** The search definition-unique name of this rank profile */
+    /**
```

Comment:
You ran some kind of automated reformatting here? Please don't do that ...

Reply:
that was not intended. Will undo.